tests/unit/Trace/Integrations/Guzzle/EventSubscriberTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Copyright 2018 OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace OpenCensus\Tests\Unit\Trace\Integrations\Guzzle;
+
+use OpenCensus\Core\Context;
+use GuzzleHttp\Client;
+use GuzzleHttp\Subscriber\Mock;
+use GuzzleHttp\Subscriber\History;
+use GuzzleHttp\Message\Response;
+use OpenCensus\Trace\Tracer;
+use OpenCensus\Trace\Exporter\ExporterInterface;
+use OpenCensus\Trace\Integrations\Guzzle\EventSubscriber;
+use Prophecy\Argument;
+
+/**
+ * @group trace
+ */
+class EventSubscriberTest extends \PHPUnit_Framework_TestCase
```

Comment:
phpunit 4.8 is very old. We should move to newer version. You can do it with another PR though.